### DISCUSSION TITLE
ScriptReference/SceneManagement.SceneManager.LoadScene

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#267] [Notes]
# ScriptReference/SceneManagement.SceneManager.LoadScene

https://docs.unity3d.com/ScriptReference/SceneManagement.SceneManager.LoadScene.html

**Comment by radiatoryang** [comment]
When loading a scene additively, the light settings (skybox, fog, etc) might conflict... to tell Unity which scene settings to use, use SceneManager.SetActiveScene (but wait at least one frame after loading the scene before setting it to be the active scene)... for more info, see [this Catlike Coding tutorial](https://catlikecoding.com/unity/tutorials/object-management/multiple-scenes/)  

```csharp
IEnumerator LoadLevel () {  
    SceneManager.LoadScene("Level 1", LoadSceneMode.Additive);  
    yield return null;  
    SceneManager.SetActiveScene(SceneManager.GetSceneByName("Level 1"));  
}
```
